<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Confirmation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #e1e7ed;
border-radius : 3px ;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="DialogTitle">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>11</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : black ; 
font-size: 25px;
font : bold ;</string>
   </property>
   <property name="text">
    <string>Delete Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="DialogLabel">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>56</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-size: 13px;
font-weight: 700;
</string>
   </property>
   <property name="text">
    <string>Are you sure you want to delete Discord ? </string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>122</y>
     <width>120</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #172133;
color: #cbd5e1;
font-weight: bold;
font-size: 13px;
}
QPushButton:hover {
    background-color: #212d40;
}</string>
   </property>
   <property name="text">
    <string> Yes, Delete</string>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/Assets/Icons/confirm.svg</normaloff>:/Assets/Icons/confirm.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>124</y>
     <width>120</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #172133;
color: #cbd5e1;
font-weight: bold;
font-size: 13px;
}
QPushButton:hover {
    background-color: #212d40;
}</string>
   </property>
   <property name="text">
    <string> No, Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/Assets/Icons/discard.svg</normaloff>:/Assets/Icons/discard.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="DialogTitleIcon">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>17</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color : black ; 
font-size: 25px;
font : bold ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
